yjit/src/yjit.rs

```diff
@@ -207,3 +207,19 @@ pub extern "C" fn rb_yjit_simulate_oom_bang(_ec: EcPtr, _ruby_self: VALUE) -> VA
 
     return Qnil;
 }
+
+/// Push a C method frame if the given PC is supposed to lazily push one.
+/// This is called from rb_raise() (at rb_exc_new_str()) and other functions
+/// that may make a method call (e.g. rb_to_int()).
+#[no_mangle]
+pub extern "C" fn rb_yjit_check_pc(pc: *mut VALUE) {
```

Comment:
Small nitpick: I feel like this method name could be more descriptive, maybe `rb_yjit_lazy_push_frame` or something like that? The current name makes it sound like it's an error or safety check.

Reply:
renamed it to `rb_yjit_lazy_push_frame` https://github.com/ruby/ruby/pull/10080/commits/b153483379036187cadbf8c24c8449d862261290